packages/app/src/background/services/validation/source.ts

```diff
@@ -1,51 +1,45 @@
-import { MerkleProofSource, MerkleProofStorageUrl } from "@cryptkeeperzk/types/dist/src/proof/zkProof";
-
-import type { IMerkleProof, IMerkleProofArtifacts, IMerkleProofInputs } from "@cryptkeeperzk/types";
+import { IMerkleProof, IMerkleProofArtifacts, IMerkleProofInputs, MerkleProofStorageUrl } from "@cryptkeeperzk/types";
```

Comment:
```suggestion
import type { IMerkleProof, IMerkleProofArtifacts, IMerkleProofInputs, MerkleProofStorageUrl } from "@cryptkeeperzk/types";
```

Reply:
I dont think we need to do this in case of imports from "@cryptkeeper/types" because we know that they are all types.

Reply:
It's not about the indication that it's the type, it's about build optimization. In production bundle we don't need these types.